datamade/bga-payroll
# Mailchimp prep

## Overview
This changes the names of some settings to coincide with the use of the Mailchimp api instead of Salsa

### Notes
For more detailed notes, see the [django-salsa-auth PR](https://github.com/datamade/django-salsa-auth/pull/16) related to this 

## Testing Instructions
- Set this up with [the mailchimp branch of django-salsa-auth](https://github.com/datamade/django-salsa-auth/tree/mailchimp):
  - Follow directions found in that repo's readme if needed
  - Add the local directory path for that repo to the app volumes in `docker-compose.yml` in the format `- <your path>:/django-salsa-auth`
  - Bring up the payroll container
- Confirm that the following processes work as expected:
  - Logging in with a non-subscribed user and getting an error message displayed
  - Signing up and subscribing a new user
    - Before going through the email verification, it may be worth trying to log in with that user to confirm that the "We've already sent an email..." message displays as expected
  - Logging in with a subscribed user and searching
